jekyll-slim.gemspec

```diff
@@ -14,11 +14,12 @@ Gem::Specification.new do |gem|
 
   gem.files         = `git ls-files`.split($/)
   gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
+  gem.test_files    = gem.files.grep(%r{^(test|spec|features|testapp)/})
   gem.require_paths = ['lib']
 
-  gem.add_runtime_dependency 'jekyll', '~> 2.0'
-  gem.add_runtime_dependency 'sliq'
+  gem.add_runtime_dependency 'byebug'
```

Comment:
debug gem should be ~~removed~~ listed with `add_development_dependency` instead. (Bundler inherits dependencies from the gemspec, and if byebug isn't specified, it's going to break someone's development workflow.)